src/mixins/filing-template-mixin.ts

```diff
@@ -691,6 +692,83 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.storeStaffPayment(filing)
   }
 
+  /**
+   * Parses a draft special resolution filing into the store.
+   * @param filing the special resolution filing
+   * @param entitySnapshot the latest entity snapshot
+   */
+  parseSpecialResolutionnFiling (filing: SpecialResolutionFilingIF, entitySnapshot: EntitySnapshotIF): void {
+    // store Entity Snapshot
+    this.setEntitySnapshot(entitySnapshot)
+
+    // store Entity Type
+    this.setEntityType(filing.alteration.business?.legalType || entitySnapshot.businessInfo.legalType)
+
+    // store Business Information
+    this.setBusinessInformation({
+      ...filing.business,
+      ...filing.alteration.business
+    })
+
+    // store Name Request data
+    this.setNameRequest(
+      filing.alteration.nameRequest ||
+      {
+        legalType: entitySnapshot.businessInfo.legalType,
+        legalName: entitySnapshot.businessInfo.legalName,
+        nrNumber: entitySnapshot.businessInfo.nrNumber
+      }
+    )
+
+    // store Provisions Removed
+    this.setProvisionsRemoved(filing.alteration.provisionsRemoved)
+
+    // store Office Addresses **from snapshot** (because we don't change office addresses in an alteration)
+    this.setOfficeAddresses(entitySnapshot.addresses)
+
+    // store People And Roles **from snapshot** (because we don't change people and roles in an alteration)
```

Comment:
More cleanup in the comments at least, and maybe functionality depending on what's needed in a Special Resolution filing.